content/docs/studio/create-view.md

```diff
@@ -13,10 +13,10 @@ To create a new view for your repository, follow these steps.
    to. ![](/img/studio/select_repo.png)
 
 4. Select the Git repository for which you want to create a view. For this,
-   hover over the required repository and click on `Connect`.
+   hover over the required repository and click `Connect`.
```

Comment:
potayto, potahto 😬

Reply:
I checked some usage SO question for this :) It's potato :))

Reply:
🍠